gravityflow/gravityflowfolders
# Add 'Move to Folder' in Admin and Bulk Actions

## Description
Re: [Issue# 68](https://github.com/gravityflow/backlog/issues/68)
This update adds the 'Move to Folder' option to move an entry from a folder to another in Admin and Bulk Actions.

## Testing instructions

- Test 'Move to Folder' in 'Admin Actions'. Ensure that the entry has successfully moved from the currently assigned 'Folder X' to the selected 'Folder Y'.
- Test 'Move to Folder' in 'Bulk Actions'. Ensure all the selected entries reflect the updated folders.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes
Some added points about these new operations on the 'Gravity Flow Folders' documentation: https://docs.gravityflow.io/article/124-folders-extension

## Checklist:
